### DISCUSSION TITLE
Upload file got mixed blocked

### DISCUSSION THREAD

**Discussion by holiq** [filamentphp/filament#15088] [Help]
### Package

filament/filament

### Package Version

v3.2.128

### Laravel Version

v11.34.2

### PHP Version

PHP 8.3.14

### Problem description

I have a custom upload file and it got mixed block with scheme http

![Screenshot from 2024-12-13 19-35-15](https://github.com/user-attachments/assets/9963854c-2737-4a12-83a6-94ae978cbed2)

### Expected behavior

can upload normally like when on local

### Steps to reproduce

my form on the ArticleResource

```php
    public static function form(Form $form): Form
    {
        return $form
            ->schema([
                Forms\Components\TextInput::make('title')
                    ->required()
                    ->maxLength(255),
                Forms\Components\Textarea::make('content')
                    ->required()
                    ->columnSpanFull(),
                Forms\Components\FileUpload::make('image')
                    ->image()
                    ->columnSpanFull()
                    ->saveUploadedFileUsing(function (TemporaryUploadedFile $file) {
                        return UploadImageAction::resolve()->execute(
                            file: $file,
                            path: 'articles',
                        );
                    }),
            ]);
    }
```

my action for handle the upload with Google Cloud Storage
```php
    public function execute(?UploadedFile $file, string $path = 'communities'): ?string
    {
        if (is_null($file)) {
            return null;
        }

        /** @var string $projectId */
        $projectId = config('gcs.project_id');

        /** @var string $key */
        $key = config('gcs.key');

        /** @var string $bucket */
        $bucket = config('gcs.bucket');

        $storage = new StorageClient([
            'keyFilePath' => public_path($key),
            'projectId' => $projectId,
        ]);

        $bucket = $storage->bucket($bucket);

        $fileName = $path.'/'.uuid_create().'.'.$file->getClientOriginalExtension();

        /** @var string $filePath */
        $filePath = fopen($file->getRealPath(), 'r');

        $bucket->upload(
            $filePath,
            [
                'name' => $fileName,
            ]
        );

        return "https://storage.googleapis.com/hidroqu/$fileName";
    }
```

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/HidroQu/HidroQu-API

### Donate 💰 to fund this issue

- You can donate funding to this issue. We receive the money once the issue is completed & confirmed by you.
- 100% of the funding will be distributed between the Filament core team to run all aspects of the project.
- Thank you in advance for helping us make maintenance sustainable!

<a href="https://polar.sh/filamentphp/filament/issues/15082">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/filamentphp/filament/issues/15082/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/filamentphp/filament/issues/15082/pledge.svg">
</picture>
</a>

**Comment by holiq** [comment]
I have to try add force https on AppServiceProvider its doesnt effect anything
```php
if($this->app->environment('production')) {
    \URL::forceScheme('https');
}
```

the app deployed on Cloud Run with Caddy as the web server
```
:3000

root * /var/www/html/public
encode gzip
php_fastcgi 127.0.0.1:9000
file_server

```

**Comment by danharrin** [comment]
Setup-specific issues are not usually bugs, and are not possible for us to debug remotely. As such, I am converting this to a discussion.
